### ISSUE TITLE
How to use the pretrained model on COCO to finetune my own dataset?

### ISSUE THREAD

**Issue by ChunmianLin** [open-mmlab/mmdetection#6968]
Hi, I want to use the pretrained model to finetune on my own data, how should I do? There occurs dimension mismatch error, i.e. coco 80 classes vs. my dataset 14 classes.
Thanks!

**Comment by RangiLyu** [comment]
Please refer to the doc: https://mmdetection.readthedocs.io/en/latest/tutorials/finetune.html

**Comment by ChunmianLin** [comment]
@RangiLyu Thanks for your reply.
Following your guidance, I carefully read the doc: https://mmdetection.readthedocs.io/en/latest/tutorials/finetune.html, but it still encounters some bugs. For instance, I want to use the QuetInst with R-101-FPN pretrained on COCO dataset, how should I do?

**Comment by ChunmianLin** [comment]
@RangiLyu My own dataset contains 14 classes, and I have modified the num_class parameters in the config file. But I do not know how to import the pretrained COCO model. Is it in the 'pretrained' item in the config file? or the '--resume-from' parameter in the terminal?

**Comment by ChunmianLin** [comment]
@RangiLyu Any helps would be appreciated a lot! Thanks!

**Comment by RangiLyu** [comment]
> For instance, I want to use the QuetInst with R-101-FPN pretrained on COCO dataset, how should I do?

Sorry for the late reply! You can add your pretrained detector model path to `load_from = ` in the config file.

**Comment by ChunmianLin** [comment]
@RangiLyu Many thanks! You mean, I can add 'load_from' parameter in the config file, to introduce the pretrained model?

**Comment by RangiLyu** [comment]
Yes. Just like the document says https://mmdetection.readthedocs.io/en/latest/tutorials/finetune.html#use-pre-trained-model

**Comment by ChunmianLin** [comment]
@RangiLyu However, due to the inherit of config file, where should I insert the 'load_from' parameter? For instance, the config file 'queryinst_r101_fpn_300_proposal_crop_mstrain_480-800_3x_coco.py' would continue to use three configs: 'queryinst_r50_fpn_300_proposals_crop_mstrain_480-800_3x_coco.py', 'queryinst_r50_fpn_100_proposals_mstrain_480-800_3x_coco.py ', and 'queryinst_r50_fpn_1x_coco.py'. I do not know where the pretrained model should be inserted?

**Comment by RangiLyu** [comment]
Just add `load_from=xxx` in the config you use. It will overwrite the base configs.

**Comment by ChunmianLin** [comment]
> Just add `load_from=xxx` in the config you use. It will overwrite the base configs.

Thanks. Following your guidance, I have loaded the pretrained COCO model to fine-tune on my own data. Part of log information is shown as follows:
<img width="938" alt="1643355486(1)" src="https://user-images.githubusercontent.com/53289159/151506907-c53cd15a-1c76-4ee4-aee1-e6f978b372c1.png">

Is it normal?
By the way, the printed loss information confuses me a lot. What does the 'grad_norm' mean? The larger value implies the better training, or vice versa? And the value of 'stage0_pos_acc' is small even though I load the pretrained model, is it normal?
<img width="914" alt="1643355800(1)" src="https://user-images.githubusercontent.com/53289159/151506964-a8e70620-ef69-4c2c-8586-c4b36f228774.png">

**Comment by ZwwWayne** [comment]
There are so many mismatched keys, which is weird. Which key do you modify? Could you post your full log?
